### ISSUE TITLE
Bug: End of houselhold creation and redirection

### ISSUE THREAD

**Issue by 19emtuck** [TomBursch/kitchenowl#596]
### Is this urgent?

No

### What parts are affected

Frontend

### What is the server version

105

### What is the client version

0.6.7

### What platform are you using

iOS, Linux

### What's the problem 🤔

User is not redirected after household entity is correctly created. This is annoying because if you don’t pay attention to spinner you don’t know if process was successfull or not. A redirection to main households list or the just created household world be great imho.

Process illustration : 

You fill the form

![image](https://github.com/user-attachments/assets/1696a247-40b3-499f-9690-89ba1cc962b5)

then spinner
![image](https://github.com/user-attachments/assets/ce5cb967-e41b-4e2f-a2b6-f6312b86746e)

then you’re stuck on the same screen
a redirect (main household or newly created entity) or a message would complete the process

![image](https://github.com/user-attachments/assets/81c4ab4e-b7f3-4522-8871-1a8999ee3493)

**Comment by TomBursch** [comment]
I just tried it on my device and the household gets opened on completion. In your tests is the household actually created?

**Comment by 19emtuck** [comment]
Yes the household is created. I get it while coming back to households list
device ios 18.1.1 
That’s just weird to stay in the create formular
